test/helpers/builder/index.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const { omit, get } = require('lodash/fp');
+const _ = require('lodash');
+const modelsUtils = require('../models');
+const { sanitizeEntity } = require('../../../packages/strapi-utils');
+const actionRegistry = require('./action-registry');
+
+const createTestBuilder = (options = {}) => {
+  const { initialState } = options;
+  const omitActions = omit('actions');
+  const getDefaultState = () => ({ actions: [], models: [], fixtures: {}, ...initialState });
```

Comment:
getDefault state should be defined outside and only return "default" state so not the initialState :) you should merge the object on `state` definition instead